frontend/taipy/src/DataNodeViewer.tsx

```diff
@@ -420,7 +417,7 @@ const DataNodeViewer = (props: DataNodeViewerProps) => {
 
     // Datanode data
     const dtType = props.data && props.data[DatanodeDataProps.type];
-    const dtValue = (props.data && props.data[DatanodeDataProps.value]) ?? ((dtType == "float") ? null : undefined);
+    const dtValue = (props.data && props.data[DatanodeDataProps.value]) ?? (dtType == "float" ? null : undefined);
```

Comment:
Is that a problem for a linter?
Because I find the previous (heavier) line more readable...

Reply:
Automatically changed by the formatter